Basic Rules.txt

```diff
@@ -544,14 +538,23 @@ Saving Throws
 
 	Saving throws are used in dire or critical situations to defend against very 
 negative effects.
-- Will: defend against a mind affect (Ex: throw off hopelessness after crashing in an escape pod)
-- Shock: maintain composure after serious damage (Ex: stay conscious after being shot by a high caliber round)
-- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy if they leap from one piece of cover to another).
+
+- Will: defend against a mind affect (Ex: throw off hopelessness after crashing 
+	in an escape pod)
+- Shock: maintain composure after serious damage (Ex: stay conscious after being 
+	shot by a high caliber round)
+- Reflex: quickly react to an event (Ex: dodge falling debris, fire at an enemy 
+	if they leap from one piece of cover to another).

```

Comment:
Blocker: This change will confuse the parser. Before it would have rendered as a stat block (which on reflection might not have been the best way to do it) the new format will render as unknowns.

Reply:
Really? I expected the leading dash to take precedence and make it read as an unordered list. Will revert.